src/components/Profile/Profile.tsx

```diff
@@ -16,41 +16,35 @@ import {
   Container,
 } from '@material-ui/core';
 import { useSelector } from 'react-redux';
-import { ReduxState } from '../../redux/combinedReducer';
+import { ReduxState } from 'redux/combinedReducer';
 
 interface ProfileProps {
   open: boolean;
   handleClose: () => void;
 }
 
 const Profile = (props: ProfileProps) => {
+  const classes = profileStyles();
+
   const user = useSelector((state: ReduxState) => {
     return state.user;
   });
-  const classes = profileStyles();
 
   const [name, setName] = React.useState(user.name);
-  const [email, setEmail] = React.useState(user.email);
   const [phone, setPhone] = React.useState(user.phone);
 
   useEffect(() => {
     setName(user.name);
-    setEmail(user.email);
     setPhone(user.phone);
-  }, [user.name, user.email, user.phone]);
+  }, [user]);
 
-  const nameInput = React.createRef();
-  const emailInput = React.createRef();
-  const phoneInput = React.createRef();
+  const nameInput = React.createRef<string>();
```

Comment:
Removed the ability for users to change email, because that shouldn't be modifyable